### ISSUE TITLE
Loss for autoencoder

### ISSUE THREAD

**Issue by qiwen-1123** [NVlabs/CMD#5]
Hi,

thanks for the great work!

I have a question regarding the loss function for training autoencoder.

In the paper, you mentioned We train our autoencoder using a simple reconstruction objective (e.g., l2 loss), but in the code
`    if mode == "autoenc":
        criterion = (
            torch.nn.MSELoss() if args.mode == 'latent'
            else LPIPSWithDiscriminator(disc_start=10000000).to(device)
            )`
you trained with the LPIPSWithDiscriminator, since you are using pixel mode. So the loss is not just a L2?
